entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_request.proto

```diff
@@ -50,3 +50,18 @@ message SetAttribute {
   ColumnIdentifier attribute = 1;
   LiteralConstant value = 2;
 }
+
+message BulkEntityArrayAttributeUpdateRequest {
+  string entityType = 1;
```

Comment:
Match buf conventions please (guessing it's not on in this repo, but we should enforce for new stuff). Here at least, camelCase vs snake_case.

Reply:
Done